package.json

```diff
@@ -76,7 +76,8 @@
         "trakt.tv-ondeck": "0.x",
         "underscore": "1.x.x",
         "upnp-mediarenderer-client": "1.x.x",
-        "urijs": "1.x.x"
+        "urijs": "1.x.x",
+        "webtorrent": "^0.94.4"
```

Comment:
yeah, we won't merge before properly reviewing, please make that PR peerflix removing =)